kits/ci-cd-diagnosis-agent/CHANGELOG.md

```
@@ -0,0 +1,18 @@
+# Changelog
+
+All notable changes to the **AgentKit CI/CD Diagnosis Agent** will be documented in this file.
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+---
+
+## [1.0.0] - 2026-07-28
+
+### Added
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)